shell/egl_window.cc

```diff
@@ -366,16 +390,8 @@ const struct wl_callback_listener EglWindow::frame_listener = {redraw};
   return nullptr;
 }
 
-void EglWindow::set_fullscreen(bool value) {
-  if (value) {
-    xdg_toplevel_set_fullscreen(m_xdg_toplevel, m_display->GetOutput());
-  } else {
-    xdg_toplevel_unset_fullscreen(m_xdg_toplevel);
-  }
-
-  // handle_xdg_surface_configure(this, m_xdg_surface, 0);
-  //  wl_callback_add_listener(wl_display_sync(m_display->GetDisplay()),
-  //                           &xdg_configure_callback_listener, this);
+void EglWindow::set_fullscreen() {
+  xdg_toplevel_set_fullscreen(m_xdg_toplevel, nullptr);
```

Comment:
You've changed this 3 times in this series. I would keep the bool as is. You might want to change from fullscreen to unfulscreen on different shells.